### PR TITLE
PWA: Improve manifest file

### PR DESCRIPTION
**Pull request by Extarys**
- I've added the language of the PWA app by getting the language setting of the instance.
- Set text direction to auto, the browser should pick up the right direction for the user (according to specs)
- Add categories.
- Add shortcut icons. iOS and some android version support those. Clicking on the PWA app icon should open a menu with those quick links.

Since this is the main manifest file, all themes will work with this.

Generated manifest file:
image

Might try to add some icons to the links eventually. Chrome-based browser and safari I believe need a web worker for PWA to work at all. Firefox have no problem with a "fully online" PWA and doesn't require a web worker.

### CODE REVIEW

File: src/Module/Manifest.php [hunk 1]
```diff
@@ -36,12 +36,15 @@ public static function rawContent(array $parameters = [])
 		$theme = DI::config()->get('system', 'theme');
 
 		$manifest = [
-			'name'        => $config->get('config', 'sitename', 'Friendica'),
-			'start_url'   => DI::baseUrl()->get(),
-			'display'     => 'standalone',
-			'description' => $config->get('config', 'info', DI::l10n()->t('A Decentralized Social Network')),
-			'short_name'  => 'Friendica',
-			'icons'       => [
+			'name'			=> $config->get('config', 'sitename', 'Friendica'),
```
Comment (MrPetovan) [inline src/Module/Manifest.php]: Ah-ha! Please don't use tabulations to align columns as the alignment (or lack thereof) then depends on the tabulation size.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Module/Manifest.php (hunk 1 above)
Comment (Extarys) [reply src/Module/Manifest.php]: RIP. Thanks I'll change them all!!! :joy:

**Comment by Extarys** [comment]
Should 		 be fixed

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Excellent, thank you!

**Pull request merged**